lib/pf/Authentication/Source/ADSource.pm

```diff
@@ -24,6 +24,7 @@ sub available_attributes {
      { value => "sAMAccountName", type => $Conditions::SUBSTRING },
      { value => "sAMAccountType", type => $Conditions::SUBSTRING },
      { value => "userAccountControl", type => $Conditions::SUBSTRING },
+     { value => "memberOf:1.2.840.113556.1.4.1941:", type => $Conditions::SUBSTRING },
```

Comment:
Doing that:
our $NESTED_GROUP = "memberOf:1.2.840.113556.1.4.1941:";
...
 { value => "memberOf:1.2.840.113556.1.4.1941:", type => $Conditions::SUBSTRING }
kill the translation.
So what do you prefer, having nested group or having memberOf:1.2.840.113556.1.4.1941: in the admin gui ?
I can also just add a pod with  this link https://msdn.microsoft.com/en-us/library/aa746475%28v=vs.85%29.aspx